gamla/functional_generic.py

```diff
@@ -665,11 +665,11 @@ async def do(x):
             await f(x)
             return x
 
-        return do
+    else:
```

Comment:
Is it a style issue or it won't work without the `else`?

Reply:
I get a pre-commit note on it, mypy or something like that